scripts/page_browse.js

```diff
@@ -9,6 +9,10 @@ var projectSelector = function() {
     ];
 };
 
+function pageTitle() {
+    document.title = proofIntData.strings.displayImageX.replace("%s", pageBrowserData.currentPage);
```

Comment:
replace does interesting things with $ when you use a string replace function as outlined here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_string_as_a_parameter
We may want to use a replace function.

Reply:
Good point. I think we will get away with it because get_page_image_param() wouldn't let $ through. But it wouldn't hurt to be extra safe.

Reply:
I think this is OK as-is. The warning about `$` is valid but I feel very confident we won't allow special characters like that in page image names.